### PR TITLE
Added test case for DuplicatedImport.

### PR DESCRIPTION
**Pull request by treenwang**
Test case for https://github.com/INRIA/spoon/issues/4783

### CODE REVIEW

File: src/test/java/spoon/test/imports/ImportTest.java [hunk 1]
```diff
@@ -26,12 +26,7 @@
 import spoon.compiler.SpoonResourceHelper;
 import spoon.experimental.CtUnresolvedImport;
 import spoon.reflect.CtModel;
-import spoon.reflect.code.CtConstructorCall;
-import spoon.reflect.code.CtInvocation;
-import spoon.reflect.code.CtLocalVariable;
-import spoon.reflect.code.CtStatement;
-import spoon.reflect.code.CtThisAccess;
-import spoon.reflect.code.CtTypeAccess;
+import spoon.reflect.code.*;
```
Comment (MartinWitt) [inline src/test/java/spoon/test/imports/ImportTest.java:29]: Could you restore the non package import statement here?

File: src/test/java/spoon/test/imports/ImportTest.java [hunk 2]
```diff
@@ -44,11 +39,7 @@
 import spoon.reflect.declaration.CtType;
 import spoon.reflect.factory.Factory;
 import spoon.reflect.factory.FactoryImpl;
-import spoon.reflect.reference.CtExecutableReference;
-import spoon.reflect.reference.CtFieldReference;
-import spoon.reflect.reference.CtPackageReference;
-import spoon.reflect.reference.CtTypeMemberWildcardImportReference;
-import spoon.reflect.reference.CtTypeReference;
+import spoon.reflect.reference.*;
```
Comment (MartinWitt) [inline src/test/java/spoon/test/imports/ImportTest.java:42]: here too

**Review by MartinWitt — CHANGES_REQUESTED** [review]

File: src/test/java/spoon/test/imports/ImportTest.java [hunk 3]
```diff
@@ -1773,4 +1767,72 @@ public void testImportsForElementsAnnotatedWithTypeUseAnnotations() {
 				.filter(ctImport -> ctImport.prettyprint().equals("import spoon.test.imports.testclasses.badimportissue3320.source.other.SomeObjectDto;"))
 				.count());
 	}
+
+	@GitHubIssue(issueNumber = 4786, fixed=false)
```
Comment (MartinWitt) [inline src/test/java/spoon/test/imports/ImportTest.java:1771]: And here the number should be `4783` or?

Comment (I-Al-Istannen) [reply src/test/java/spoon/test/imports/ImportTest.java:1771]: That was my mistake, I picked the wrong issue, sorry :)

**Comment by slarse** [comment]
Closing this as stale, feel free to reopen and complete at a later time.

**Pull request closed without merging**